<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Менеджер пакета Gamebase</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="mainTab">
         <attribute name="title">
          <string>Главная</string>
         </attribute>
         <layout class="QGridLayout" name="mainTabLayout"/>
        </widget>
        <widget class="QWidget" name="settingsTab">
         <attribute name="title">
          <string>Настройки</string>
         </attribute>
         <layout class="QGridLayout" name="settingsLayout"/>
        </widget>
        <widget class="QWidget" name="librariesTab">
         <attribute name="title">
          <string>Библиотеки</string>
         </attribute>
         <layout class="QGridLayout" name="librariesLayout"/>
        </widget>
        <widget class="QWidget" name="appsTab">
         <attribute name="title">
          <string>Приложения</string>
         </attribute>
         <layout class="QGridLayout" name="appsLayout"/>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="statusTip">
    <string>Закрыть приложение</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>О программе...</string>
   </property>
   <property name="statusTip">
    <string>Информация о приложении</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
